tiny-web-page/style.css

```diff
@@ -19,3 +19,7 @@ h1:before {
   background-repeat: no-repeat;
   background-size: cover;
 }
+
+li{
```

Comment:
Would be even better if there was a space between the `li` and the `{`. Nicely-formatted code is easier to read and maintain :-)